stan/math/opencl/prim/bernoulli_logit_glm_lpmf.hpp

```diff
@@ -43,30 +49,29 @@ namespace math {
  * @throw std::invalid_argument if container sizes mismatch.
  */
 
-template <bool propto, typename T_alpha, typename T_beta>
-return_type_t<T_alpha, T_beta> bernoulli_logit_glm_lpmf(
-    const matrix_cl<int>& y_cl, const matrix_cl<double>& x_cl,
-    const T_alpha& alpha, const T_beta& beta) {
+template <bool propto, typename T_x, typename T_y, typename T_alpha,
+          typename T_beta,
+          require_all_prim_or_rev_kernel_expression_t<T_y, T_x, T_alpha,
+                                                      T_beta>* = nullptr>
+return_type_t<T_x, T_alpha, T_beta> bernoulli_logit_glm_lpmf(
```

Comment:
No `T_y` here?

Reply:
Yep. Scalar in `T_y` is `int` (can't be var).